oshdb-util/src/test/java/org/heigit/ohsome/oshdb/util/geometry/osmtestdata/OSHDBGeometryBuilderTestOsmTestData7xxTest.java

```diff
@@ -579,12 +579,11 @@ public void test745() {
 
   @Test
   public void test746() {
-    // IImpossible multipolygon out of two way.
+    // Impossible multipolygon out of two ways.
     OSMEntity entity1 = testData.relations().get(746900L).get(0);
     try {
       Geometry result = OSHDBGeometryBuilder.getGeometry(entity1, timestamp, tagInterpreter);
       assertTrue(result instanceof GeometryCollection || result instanceof Polygonal);
-      assertEquals(2, result.getNumGeometries());
```

Comment:
What is the expected outcome of this test? Shouldn't we test for a proper result? Or is this already covered by another test?

Reply:
The point is that in all tests with invalid input data, we basically don't care what _exactly_ comes out as the result, as long as it is anything _reasonable_ (here: not `null`, either polyon or geometry collection) and no exceptions are thrown.

Not sure why the `numGeometries` was tested here before – I mean, it makes some sense if the result is a GeometryCollection (as was returned before this PR), but if one decides to build a Polygon out of the invalid input data, it's kinda hard to construct something with two parts. :innocent: